### ISSUE TITLE
Can I get UDP traffic in proxytunnel which is passed through the ssh tunnel and how to process it?

### ISSUE THREAD

**Issue by noman720** [noman720/proxytunnel#2]
Hi @sigio If I send UDP traffic (converting UDP packet to TCP) over ssh tunnel, then there is any way to extract it in proxytunnel? Please can you give me some idea to do that?

**Comment by sigio** [comment]
I don't think so.... it's just a transparant socket... doesn't do anything after initial tunnel negotiation.
